### PR TITLE
Feature - use redis for session management

### PR DESCRIPTION
**Pull request by joefitter**
- check webdriver cookie for sessionId
- parse this
- remove all the memory store stuf
- start and stop redis before and after testsf

### CODE REVIEW

**Comment by easternbloc** [comment]
Can you please add some description to the PR to explain why you've done this. So future devs can follow your train of thought. 💘

**Comment by joefitter** [comment]
NO

**Comment by joefitter** [comment]
I actually don't want this to be merged - it was more just to see how it builds

**Comment by joefitter** [comment]
It'll all be moved to SO ACCEPTANCE

**Comment by joefitter** [comment]
which will have a lovely PR with loads of description and stuffs

**Comment by easternbloc** [comment]
then I forgive you. But the concept looks sound

File: acceptance_tests/helpers/session.js [hunk 1]
```diff
@@ -2,20 +2,52 @@
...
-const session = require('../session');
+const expressSession = require('express-session');
+const RedisStore = require('connect-redis')(expressSession);
+const cookieParser = require('cookie-parser');
 
-const SESSION_ID = 'fakeId';
-const SESSION_KEY = 'hmpo-wizard';
+const SESSION_KEY_PREFIX = 'hmpo-wizard';
+const SESSION_KEY_COOKIE_NAME = 'hod.sid';
+const SECRET = 'changethis';
```
Comment (JoeChapman) [inline acceptance_tests/helpers/session.js]: "changethis"

Comment (joefitter) [inline acceptance_tests/helpers/session.js]: eh?

**Comment by bendiggle** [comment]
Wooop! Tests are actually running and passing! LGTM

**Comment by joefitter** [comment]
👏

**Pull request merged**